Makefile.am

```diff
@@ -1688,6 +1691,21 @@ sss_ssh_knownhostsproxy_LDADD = \
     $(CLIENT_LIBS) $(TALLOC_LIBS) $(POPT_LIBS)
 endif
 
+if HAVE_SYSTEMD_UNIT
+sssd_check_socket_activated_responders_SOURCES = \
+	src/tools/sssd_check_socket_activated_responders.c
+
+sssd_check_socket_activated_responders_LDADD = \
+	$(SSSD_INTERNAL_LTLIBS) \
+	$(LTLIBINTL) \
+	$(TALLOC_LIBS) \
+	$(POPT_LIBS) \
+	$(INI_CONFIG_LIBS)
+
+sssd_check_socket_activated_responders_CFLAGS = \
+	$(AM_CFLAGS)
+endif
```

Comment:
* Please do not use tabs.
* something_CFLAGS is between _SOURCES and _LDADD for other programs
* Pleasee add `$(NULL)` at the end of lists